<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>avenuedialog</class>
 <widget class="QDialog" name="avenuedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>609</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
    <underline>true</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Avenue</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo1.png</normaloff>logo1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-image: url(./user.png);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent;
    color:white;
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox::enabled{

color:white;
font-weight: bold;
}
QLabel{color:white;}
QGroupBox::disabled{

color:gray;
font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>Nombre des lampes dans chaque avenue</string>
   </property>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>251</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView { background-color: rgba(255, 255, 255, 0); border: 1px solid gray; }
QTableView  { border: 2px solid white; border-style: outset; }
QTableView  { font-weight: bold; color: white; }
QTableView QHeaderView::section {
    background-color: transparent;
    border: 0;
    border-bottom: 1px solid white;
}
QTableView {
  gridline-color: white;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nbr">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::enabled { background-color: rgba(0, 0, 0, 0);
color:white;
 }
QPushButton { border: 2px solid white; border-radius: 5px; font-size: 12px; }
QPushButton{ border: 2px solid white; border-radius: 10px; background-color: transparent; padding: 5px;
font-weight: bold; }
QPushButton:hover { border: 2px solid white; background-color: rgba(220, 220, 220, 0.5); }
QPushButton:enabled {
  color: white;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{

color:white;
font-weight: bold;
}
QLabel{color:white;}</string>
   </property>
   <property name="title">
    <string>Table Avenue</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView { background-color: rgba(255, 255, 255, 0); border: 1px solid gray; }
QTableView  { border: 2px solid white; border-style: outset; }
QTableView  { font-weight: bold; color: white; }
QTableView QHeaderView::section {
    background-color: transparent;
    border: 0;
    border-bottom: 1px solid white;
}
QTableView {
  gridline-color: white;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: rgba(0, 0, 0, 0);
color:white;
 }
QPushButton { border: 2px solid white; border-radius: 5px; font-size: 12px; }
QPushButton { border: 2px solid white; border-radius: 10px; background-color: transparent; padding: 5px;
font-weight: bold; }
QPushButton:hover { border: 2px solid white; background-color: rgba(220, 220, 220, 0.5); }
button:enabled {
  color: white;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>401</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{

color:white;
font-weight: bold;
}
QLabel{color:white;}
QPushButton::Disabled {
  color: Gray;
}</string>
   </property>
   <property name="title">
    <string>Ajouter Avenue</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_2Nom">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: rgba(255, 255, 255, 0); border: 1px solid gray; }
QLineEdit { border: 2px solid white; border-style: outset; }
QLineEdit { font-weight: bold; color: white; }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nom                   :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border: 2px solid white; border-radius: 5px; font-size: 12px; }
QPushButton { border: 2px solid white; border-radius: 10px; background-color: transparent; padding: 5px;
font-weight: bold; }
QPushButton:hover { border: 2px solid white; background-color: rgba(220, 220, 220, 0.5); }
QPushButton:enabled {
  color: white;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: rgba(0, 0, 0, 0);

 }
QPushButton { border: 2px solid white; border-radius: 5px; font-size: 12px; }
QPushButton { border: 2px solid white; border-radius: 10px; background-color: transparent; padding: 5px;
font-weight: bold; }
QPushButton:hover { border: 2px solid white; background-color: rgba(220, 220, 220, 0.5); }
button:enabled {
  color: white;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Init</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>371</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{

color:white;
font-weight: bold;
}
QLabel{color:white;}
QPushButton::Disabled {
  color: Gray;
}</string>
   </property>
   <property name="title">
    <string>Supprimer,Modifier Avenue</string>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton { border: 2px solid white; border-radius: 5px; font-size: 12px; }
QPushButton { border: 2px solid white; border-radius: 10px; background-color: transparent; padding: 5px;
font-weight: bold; }
QPushButton:hover { border: 2px solid white; background-color: rgba(220, 220, 220, 0.5); }
QPushButton:enabled {
  color: white;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border: 2px solid white; border-radius: 5px; font-size: 12px; }
QPushButton { border: 2px solid white; border-radius: 10px; background-color: transparent; padding: 5px;
font-weight: bold; }
QPushButton:hover { border: 2px solid white; background-color: rgba(220, 220, 220, 0.5); }
QPushButton:enabled {
  color: white;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: rgba(255, 255, 255, 0.8); border: 1px solid gray; }
QLineEdit { border: 2px solid white; border-style: outset; }
QLineEdit { font-weight: bold; color: Gray; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ID      :</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>15</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color:white;}</string>
   </property>
   <property name="text">
    <string>Avenue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
